README.md

```diff
@@ -23,7 +23,7 @@ Since RxAndroid is part of the RxJava family the communication channels are simi
 compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
 // Because RxAndroid releases are few and far between, it is recommended you also
 // explicitly depend on RxJava's latest version for bug fixes and new features.
-compile 'io.reactivex.rxjava2:rxjava:2.1.8'
+compile 'io.reactivex.rxjava2:rxjava:2.x.x' // see https://github.com/ReactiveX/RxJava/releases for latest 2.x.x release
```

Comment:
Comment goes behind right end of the screen due to codeblock formatting

Can you try to play with it to achieve better readability? Maybe add it on line above :)